UnoCheck/Manifest/Gtk.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Newtonsoft.Json;
+
+namespace DotNetCheck.Manifest
+{
+	public partial class Gtk
```

Comment:
partial ?

Reply:
All classes are marked as partial. Can I remove it anyway?

![image](https://user-images.githubusercontent.com/5515535/190525843-7c008e62-8e18-4028-a18c-496d9218d2db.png)

Reply:
Let's keep it partial for now, it does not change much.